larpandoracontent/LArHelpers/LArMCParticleHelper.h

```diff
@@ -75,6 +76,11 @@ class LArMCParticleHelper
      */
     static bool IsBeamParticle(const pandora::MCParticle *const pMCParticle);
 
+    /**
+     *  @brief  Returns true if passed a leading beam MCParticle
+     */
+    static bool IsLeadingBeamParticle(const pandora::MCParticle *const pMCParticle);
```

Comment:
Should provide a good definition of leading beam particle in the header file. It's subtly different to a primary beam particle, and the primary definition could be refreshed too.